packages/utilities/src/lib/hasAtLeastOneKeyInObject.ts

```diff
@@ -0,0 +1,22 @@
+import { isNullish } from './isNullOrUndefined';
+import type { NonNullObject } from './types';
+
+/**
+ * Checks whether any of the {@link keys} are in the {@link object}
+ * @param obj - The object to check
```

Comment:
i doubt this link tag will work

Reply:
Mostly, just had to fix the `object` -> `obj`